src/components/Project.tsx

```diff
@@ -159,7 +160,7 @@ const Project = () => {
    */
   const getAllProjects = async () => {
     axios
-      .get("http://3.236.213.150:8081/projects/portfolio/all/"+cookie["portfolio"].id)
+      .get(url + "/projects/portfolio/all/"+cookie["portfolio"].id)
```

Comment:
@ddinh please store `url + "/projects"` to a variable and pass here to minimize code duplication throughout this file